_includes/v23.1/known-limitations/cdc.md

```diff
@@ -2,7 +2,6 @@
 - Changefeed target options are limited to tables and [column families](changefeeds-on-tables-with-column-families.html). [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/73435)
 - Using a [cloud storage sink](changefeed-sinks.html#cloud-storage-sink) only works with `JSON` and emits [newline-delimited json](http://ndjson.org) files. [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/73432)
 - Webhook sinks only support HTTPS. Use the [`insecure_tls_skip_verify`](create-changefeed.html#tls-skip-verify) parameter when testing to disable certificate verification; however, this still requires HTTPS and certificates. [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/73431)
-- [Webhook sinks](changefeed-sinks.html#webhook-sink) and [Google Cloud Pub/Sub sinks](changefeed-sinks.html#google-cloud-pub-sub) only have support for emitting `JSON`. [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/73432)
+- [Webhook sinks](changefeed-sinks.html#webhook-sink) only has support for emitting `JSON`. [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/73432)
```

Comment:
I believe that cloud storage can now emit with CSV too, correct?

Reply:
Correct! (If you have a recent build handy, you can test what's supported with cloud storage by using `INTO 'nodelocal://1/feedout'` as the sink URI).

Reply:
Ah great thanks! re the URI

Reply:
Suggestion: "Webhook sinks only support emitting JSON"